### ISSUE TITLE
Error on $ php switch php-...

### ISSUE THREAD

**Issue by thehawk970** [jubianchi/phpswitch#19]
I have an error when i try to switch to an personal version like php switch php-5.4.10

i have : 
PHP Warning:  file_put_contents(/home/.phpswitch.yml): failed to open stream: Permission denied in /usr/local/lib/phpswitch/src/jubianchi/PhpSwitch/Config/Dumper.php on line 31
PHP Warning:  file_put_contents(/home/.phpswitch.yml): failed to open stream: Permission denied in /usr/local/lib/phpswitch/src/jubianchi/PhpSwitch/Config/Dumper.php on line 31

May be you could make an verification and an beautiful exception :angel: 

Thehawk_

**Comment by jubianchi** [comment]
You are right, a beautifull exception or at least a warning saying that the configuration could not be dumped ;)

Thanks for the report :+1:
